package.json

```diff
@@ -53,12 +53,28 @@
     "eslint": "^6.8.0",
     "eslint-config-react-app": "^5.0.1",
     "eslint-config-react-tools": "^1.1.7",
+    "eslint-plugin-flowtype": "^4.7.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.17.0",
     "eslint-plugin-react-hooks": "^2.3.0",
+    "husky": "^4.2.3",
     "jest": "^24.9.0",
+    "lint-staged": "^10.0.9",
     "prettier": "^1.19.1",
     "serve": "^11.1.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "components/**/*.js": [
+      "eslint --fix"
+    ],
+    "lib/**/*.js": [
+      "eslint --fix"
+    ]
```

Comment:
Should we also add `pages`?

Reply:
While `pages` won't have much markup, sounds good to add it. 👍